build.psm1

```diff
@@ -386,7 +386,7 @@ Fix steps:
             # Relative paths do not work well if cwd is not changed to project
```

Comment:
@SteveL-MSFT, your last commit had 9 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

TestStartJobThrowTerminatingException
```powershell
Assert.Throws() Failure
Expected: typeof(System.Management.Automation.CmdletInvocationException)
Actual:   typeof(System.Management.Automation.Runspaces.PSSnapInException): Cannot load PowerShell snap-in Microsoft.PowerShell.Management because of the following error: Could not load file or assembly 'D:\a\1\s\test\xUnit\bin\Release\netcoreapp3.0\Microsoft.PowerShell.Commands'. The system cannot find the file specified.
---- System.Management.Automation.Runspaces.PSSnapInException : Cannot load PowerShell snap-in Microsoft.PowerShell.Management because of the following error: Could not load file or assembly 'D:\a\1\s\test\xUnit\bin\Release\netcoreapp3.0\Microsoft.PowerShell.Commands'. The system cannot find the file specified.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1538
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(PSHost host) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 68
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 45
   at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 5392
   at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 4529
   at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 4724
   at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 2413
   at System.Management.Automation.PowerShell.Invoke() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 2289
   at PSTests.Sequential.PowerShellHostingScenario.<>c__DisplayClass0_0.<TestStartJobThrowTerminatingException>b__0() in D:\a\1\s\test\xUnit\csharp\test_PowerShellAPI.cs:line 18
----- Inner Stack Trace -----
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1538
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(PSHost host) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 68
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 45
   at System.Management.Automation.PowerShell.Worker.CreateRunspaceIfNeededAndDoWork(Runspace rsToUse, Boolean isSync) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 5392
   at System.Management.Automation.PowerShell.CoreInvokeHelper[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 4529
   at System.Management.Automation.PowerShell.CoreInvoke[TInput,TOutput](PSDataCollection`1 input, PSDataCollection`1 output, PSInvocationSettings settings) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 4724
   at System.Management.Automation.PowerShell.Invoke(IEnumerable input, PSInvocationSettings settings) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 2413
   at System.Management.Automation.PowerShell.Invoke() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\PowerShell.cs:line 2289
   at PSTests.Sequential.PowerShellHostingScenario.<>c__DisplayClass0_0.<TestStartJobThrowTerminatingException>b__0() in D:\a\1\s\test\xUnit\csharp\test_PowerShellAPI.cs:line 18
```
TestRunspaceSetBreakpoints
```powershell
System.Management.Automation.Runspaces.PSSnapInException : Cannot load PowerShell snap-in Microsoft.PowerShell.Management because of the following error: Could not load file or assembly 'D:\a\1\s\test\xUnit\bin\Release\netcoreapp3.0\Microsoft.PowerShell.Commands'. The system cannot find the file specified.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1538
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(PSHost host) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 68
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 45
   at PSTests.Sequential.RunspaceTests.TestRunspaceSetBreakpoints() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 108
```
TestRunspaceWithPowerShellAndInitialSessionState
```powershell
System.Management.Automation.Runspaces.PSSnapInException : Cannot load PowerShell snap-in Microsoft.PowerShell.Management because of the following error: Could not load file or assembly 'D:\a\1\s\test\xUnit\bin\Release\netcoreapp3.0\Microsoft.PowerShell.Commands'. The system cannot find the file specified.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1538
   at PSTests.Sequential.RunspaceTests.TestRunspaceWithPowerShellAndInitialSessionState() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 73
```
TestRunspaceWithPipeline
```powershell
System.Management.Automation.Runspaces.PSSnapInException : Cannot load PowerShell snap-in Microsoft.PowerShell.Management because of the following error: Could not load file or assembly 'D:\a\1\s\test\xUnit\bin\Release\netcoreapp3.0\Microsoft.PowerShell.Commands'. The system cannot find the file specified.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1538
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(PSHost host) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 68
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 45
   at PSTests.Sequential.RunspaceTests.TestRunspaceWithPipeline() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 22
```
TestRunspaceWithPowerShell
```powershell
System.Management.Automation.Runspaces.PSSnapInException : Cannot load PowerShell snap-in Microsoft.PowerShell.Management because of the following error: Could not load file or assembly 'D:\a\1\s\test\xUnit\bin\Release\netcoreapp3.0\Microsoft.PowerShell.Commands'. The system cannot find the file specified.
   at System.Management.Automation.Runspaces.InitialSessionState.CreateDefault() in D:\a\1\s\src\System.Management.Automation\engine\InitialSessionState.cs:line 1538
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace(PSHost host) in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 68
   at System.Management.Automation.Runspaces.RunspaceFactory.CreateRunspace() in D:\a\1\s\src\System.Management.Automation\engine\hostifaces\ConnectionFactory.cs:line 45
   at PSTests.Sequential.RunspaceTests.TestRunspaceWithPowerShell() in D:\a\1\s\test\xUnit\csharp\test_Runspace.cs:line 45
```